src/test/java/api/loans/OverrideCheckOutByBarcodeTests.java

```diff
@@ -1,32 +1,43 @@
 package api.loans;
 
+import static api.support.APITestContext.getOkapiHeadersFromContext;
```

Comment:
Let's move our "new" override tests to the CheckOutByBarcodeTests class. Also, do we need to change the "old" override tests?